config/sample_config_complex.txt

```diff
@@ -0,0 +1,471 @@
+# Sample configuration file for evcouplings complex protein prediction pipeline.
```

Comment:
Please go through this at the very end and clear up (removing proteins so it is clean to start, like monomer one)

Also rename the o2 monomer config to sample_config_monomer_o2.txt